### PR TITLE
chore: Diff-Tool/Staging Directory 

### PR DESCRIPTION
**Pull request by ddixit14**
To run the diff-tool, simple checkout this branch on your local, give any execute permissions needed for the scripts, and run the ./generation/diff_tool.sh.
It will create two text files, `diffSummary.txt` which contains diff failure (diffs found), and `matchSummary.txt` which contains diff success (no diff found).
Below are the files created after a full run ->
[diffSummary.txt](https://github.com/googleapis/google-cloud-java/files/9626515/diffSummary.txt)
[matchSummary.txt](https://github.com/googleapis/google-cloud-java/files/9626529/matchSummary.txt)

### CODE REVIEW

**Comment by ddixit14** [comment]
(Please do not delete this PR, I am using this branch for staging process)

**Comment by suztomo** [comment]
> give any execute permissions needed for the scripts

The user of the script should not do this. You do that when you commit files to repository.

**Comment by suztomo** [comment]
> This branch has conflicts that must be resolved

**Comment by suztomo** [comment]
@ddixit14 Can you clarify the intention of this pull request? Do you intend to merge this pull request once approved? I see ~ 800 pom.xml files are modified here.

**Comment by ddixit14** [comment]
> @ddixit14 Can you clarify the intention of this pull request? Do you intend to merge this pull request once approved? I see ~ 800 pom.xml files are modified here.

No, this should not be merged. I can merge the three diff scripts separately. This branch is entirely to stage directories, and then run the diff_tool to check for differences. Once we are confident that we are good to release from monorepo, I will merge only the three diff scripts. Diff-scripts need a branch which does not have any snapshot versions in it, so a release-please PR branch would be the best candidate going forward.

File: generation/diff_jar.sh
Comment (suztomo) [inline generation/diff_jar.sh:12]: How about introducing a flag? Such as "COMPARE_SIZE" environment variable.

File: generation/diff_tool.sh
Comment (suztomo) [inline generation/diff_tool.sh:11]: Add how to get these values.

File: generation/diff_tool.sh
Comment (suztomo) [inline generation/diff_tool.sh:3]: Add output of this script.

Comment (suztomo) [reply generation/diff_tool.sh:3]: Explain environmental variable to toggle size check.

File: .kokoro/release/publish_javadoc11.sh
Comment (suztomo) [inline .kokoro/release/publish_javadoc11.sh:52]: The changes in this .kokoro/release/publish_javadoc11.sh is irrelevant to diff-tool, right?

File: .kokoro/release/common.cfg
Comment (suztomo) [inline .kokoro/release/common.cfg:12]: Create a pull request that only focuses this change.

**Review by suztomo — APPROVED** [review]

**Review by meltsufin — COMMENTED** [review]
Can we remove all the non `diff_*.sh` files from this PR, and merge this in?

**Pull request closed without merging**
